klpbuild/config.py

```diff
@@ -95,7 +92,6 @@ def setup_user_env(self, basedir):
 
     def load_user_conf(self):
         config = configparser.ConfigParser()
-        logging.info(f"Loading user configuration from '{self.user_conf_file}'")
```

Comment:
This is kind of useful, no? I know that due to the current design we get the message several times per run, but still.